lib/http/request/body.rb

```diff
@@ -32,14 +29,10 @@ def size
       #
       # @yieldparam [String]
       def each
-        return enum_for(__method__) unless block_given?
```

Comment:
I removed this because we don't need it anymore, and because how JRuby implements Enumerators (https://github.com/jruby/jruby/issues/4583) it's not usable on JRuby when fetching next element mutates the previous one (reading file with a buffer string). So I thought it's better to remove it if it's not being used.